docs/openapi-docs/src/main/scala/sttp/tapir/docs/openapi/EndpointToOperationResponse.scala

```diff
@@ -97,6 +106,17 @@ private[openapi] class EndpointToOperationResponse(
     }
   }
 
+  private def mergeMediaTypesToAnyOf(bodies: Vector[MediaType]): MediaType =
+    bodies.toSet.toList match {
+      case List(body) => body
+      case bodies =>
+        MediaType(
+          schema = Some(ASchema(anyOf = bodies.flatMap(_.schema))),
+          example = bodies.flatMap(_.example).headOption,
+          examples = bodies.flatMap(_.examples).toListMap
```

Comment:
The [spec](https://spec.openapis.org/oas/v3.1.0#fixed-fields-11) says that `example` and `examples` fields should be mutually exclusive.